src/Screens/NewWorkshop.js

```
@@ -339,102 +439,118 @@ function NewWorkshop() {
-                style={{ color: isDarkModeOn ? 'white' : 'black' }}
-                className="description-box"
-                dangerouslySetInnerHTML={{ __html: workshopData?.description || "Workshop Description" }}
-              />
+                </Grid>
+
+                {/* Row 4: Description */}
+                <Grid item xs={12}>
+                  <div
+                    style={{ color: isDarkModeOn ? "white" : "black" }}
+                    className="description-box"
+                    dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)